client/src/charts/wrapped_nivo/wrapped_nivo_common.yaml

```diff
@@ -3,5 +3,5 @@ show_table:
   fr: Afficher le tableau de données
 
 default_table_name:
-  en: Table for graph
-  fr: Tableau pour graphique
+  en: Table of data
```

Comment:
#598 

This was originally Table of data. Changed it because it was a weak name...
Although table of graph was a pretty weak name in itself lol

Not sure how we should go about this

Reply:
😂 so my rationale for changing it to "Table of data" is that, in a11y, screen reader will read this table name first. But there isn't actually a graph for in a11y so it didn't really made sense